crates/fuel-core/src/database/sealed_block.rs

```diff
@@ -51,6 +54,23 @@ impl OnChainIterableKeyValueView {
         }
     }
 
+    /// Returns a `Block`, which is sealed, by height
+    pub fn get_block_by_height(
+        &self,
+        height: &BlockHeight,
+    ) -> StorageResult<Option<Block>> {
+        if self
```

Comment:
Why do we want to do an additional lookup into `SealedBlockConsensus` table? We can go directly to the `self.get_full_block`

Reply:
addressed in 5498c5fd9